### DISCUSSION TITLE
Converting Gene expression database (human) to .CSV or other formats

### DISCUSSION THREAD

**Discussion by Mwilaputa** [Open-Systems-Pharmacology/Forum#1573] [Questions & Problems]
Is there a way converting gene expression database from .expressionDb to .CSV or other file formats?

**Comment by msevestre** [comment]
No. What are you trying to achieve?

**Reply by Mwilaputa** [reply]
I wanted to select specific gene/enzymes responsible for metabolism certain drugs
